packages/smooth_app/test/users/login_page_layout_test.dart

```diff
@@ -0,0 +1,70 @@
+import 'dart:io';
+
+import 'package:flutter/services.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:openfoodfacts/personalized_search/product_preferences_selection.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:smooth_app/data_models/product_preferences.dart';
+import 'package:smooth_app/data_models/user_management_provider.dart';
+import 'package:smooth_app/data_models/user_preferences.dart';
+import 'package:smooth_app/pages/user_management/login_page.dart';
+import 'package:smooth_app/themes/theme_provider.dart';
+
+import '../tests_utils/goldens.dart';
+import '../tests_utils/mocks.dart';
+
+void main() {
+  group('LoginPage looks as expected', () {
+    for (final bool themeDark in <bool>[true, false]) {
+      final String theme = themeDark ? 'Dark' : 'Light';
+
+      testWidgets(theme, (WidgetTester tester) async {
+        final HttpOverrides? priorOverrides = HttpOverrides.current;
```

Comment:
I don't think you are mocking HTTP requests here, u can avoid em

Reply:
Sometimes the app doesn't perform as expected when offline, I didn't wanted to take chances.

Reply:
HTTP calls by default is not allowed in testing, so the mocking here is for another stuff, unless you define you own mocks you can simply omit them here, give it a try and let me know if it really caused any problem when u removed these lines